helm/boldreports/values.yaml

```diff
@@ -0,0 +1,120 @@
+namespace: bold-services
+appBaseUrl: ''
+optionalLibs: ''
+clusterProvider: ''
+
+persistentVolume:
+  # persistent volumes were global resources. 
+  # so if you already have Bold Reports installed in your cluster, then the previous persistent volume name will conflict with current installation.
+  # Change this name to avoid conflicts with previous Bold Reports persistent volumes.
+  name: bold-services-fileserver
+  capacity: 3Gi
+  gke:
+    fileShareName: ''
+    fileShareIp: ''
+  eks:
+    efsFileSystemId: ''
+  aks:
+    fileShareName: ''
+    # base64 string
+    azureStorageAccountName: ''
+    # base64 string
+    azureStorageAccountKey: ''
+  onpremise:
+    hostPath: ''
+    
+image:
+  repo: gcr.io/boldreports
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 5.3.8
+  pullPolicy: IfNotPresent
+imagePullSecrets: []
+
+loadBalancer:
+  type: nginx
+  affinityCookie:
+    enable: true
+    affinityCookieExpiration: 600
+  # For Kubernetes >= 1.18 you should specify the ingress-controller via the field ingressClassName
+  # See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#specifying-the-class-of-an-ingress
+  # ingressClassName: nginx
+  singleHost: 
+    secretName: boldreports-tls
+  # multipleHost:
+    # hostArray:
+      # - hosts: 
+          # - kubernetes.docker.internal
+          # - example.com
+        # secretName: boldreports-tls
+
+instanceConfig:
+  - app: id-web
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: id-api
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: id-ums
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: reports-web
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: reports-api
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: reports-jobs
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: reports-reportservice
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+  - app: reports-viewer
+    replicaCount: 1
+    minReplicas: 1
+    maxReplicas: 20
+    cpuResourceRequests: 250m
+    memoryResourceRequests: 750Mi
+    
+service:
+  type: ClusterIP
+  port: 80
+  
+autoscaling:
+  enabled: true
+  targetCPUUtilizationPercentage: 80
+  targetMemoryUtilizationPercentage: 80
+  behavior:
+    stabilizationWindowSeconds: 60
+    podsValue: 1
+    podsPeriodSeconds: 60
+    percentValue: 10
+    percentPeriodSeconds: 60
+    
+bingMapWidget:
```

Comment:
remove this bingmapwidget section in corresponding places. we didn't use this.

Reply:
I have removed the section